app-text/nfoview/nfoview-1.28.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=no
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1 xdg-utils
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/otsaloma/nfoview.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/otsaloma/nfoview/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Simple viewer for NFO files, which are ASCII art in the CP437 codepage"
+HOMEPAGE="https://otsaloma.io/nfoview/"
+
+LICENSE="GPL-3+"
```

Comment:
upstream license looks to be standard `GPL-3`. Why the +?

Reply:
sourcefiles say

```
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
```

Reply:
Fair enough.
Interestingly, github still calls it GPL-3.

Reply:
@epsilon-0 the Github web interface is really bad at detecting licenses. It doesn't show the `+` icon anywhere and fails to detect if multiple licenses are used in a package. You always need to check headers for source files, etc, need to dig a little bit deeper. There are also 3rd party tools to detect licenses for a package.

Reply:
Ah, makes sense.  
Note for future.